### ISSUE TITLE
ACE PyTorch with LAMMPS

### ISSUE THREAD

**Issue by Basti1211** [FitSNAP/FitSNAP#157]
Hello there,

I have tried to fit your example potential "Ta_PACE_PyTorch_NN", which works well - now I want to test the potential in LAMMPS. 

I tried to follow the input script for LAMMPS of the example "Ta_PyTorch_NN" and modify it for ACE capabilities. Therefore, I need to set the potential file AND the descriptor. I have the potential file, but I need help figuring out what the descriptor style and file should look like. Is it possible to fit an NN-Potential with ACE descriptors, and if so, which files should I use?

Best,
Bastian

**Comment by rohskopf** [comment]
Hi @Basti1211 , 

Thanks for pointing this out.

In LAMMPS, PyTorch models are used in the ML-IAP package like you've seen. Currently we don't have ACE descriptors in the ML-IAP package. We therefore can't do MD with ACE PyTorch models yet.

This is definitely something we're interested in adding to LAMMPS. We compute the ACE descriptors and their gradients in [`compute pace`](https://github.com/jmgoff/lammps_compute_PACE/blob/compute-pace/src/ML-PACE/compute_pace.cpp) for fitting. Now we just gotta plug those into the PyTorch model to get energies and forces during MD. The ML-IAP package handles that descriptor/model interface. 

I'll leave this issue open and keep you posted.

Drew

**Comment by Basti1211** [comment]
Hi Drew,

Thank you for your fast answer and for keeping me posted!

Best, 
Bastian

**Comment by mitwood** [comment]
Hey @jmgoff has there been there an update to the ACE MLIAP package in LAMMPS that would change the status of this issue?

**Comment by jmgoff** [comment]
Yes, MD can now be done with ACE pytorch neural networks in LAMMPS in the current development branch. It may also be in the current stable release. If not, should be in the upcoming stable release.

@Basti1211  , if you want examples on how to run the ACE pytorch neural networks, you may find one in the mliap examples for lammps now:
https://github.com/lammps/lammps/blob/develop/examples/mliap/in.mliap.pytorch.ace.NN
